common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -1052,9 +1052,8 @@ def setUp(self):
         self.discussion_page = InlineDiscussionPage(self.browser, self.discussion_id)
         self.additional_discussion_page = InlineDiscussionPage(self.browser, self.additional_discussion_id)
 
-    # This test is too flaky to run at all. TNL-6215
     @attr('a11y')
-    @nottest
+    @pytest.mark.skip(reason='This test is too flaky to run at all. TNL-6215')
```

Comment:
I know it's not necessarily the goal behind this PR, but shouldn't we just remove the test if its too flaky to run?

Reply:
Sure, I'll create a separate PR for that and link it to [EDUCATOR-275](https://openedx.atlassian.net/browse/EDUCATOR-275).